src/main/java/org/terracotta/build/plugins/AngelaPlugin.java

```diff
@@ -116,6 +121,20 @@ public void execute(Task t) {
           } else {
             task.systemProperty("angela.kitInstallationDir", customKitDir.get().getAsFile().getAbsolutePath());
           }
+          String customHostNames = angelaPluginExtension.getCustomHosts().getOrNull();
+          if (customHostNames != null) {
+            File file = new File(fakeHostDir.get().getAsFile(), "fake-hosts.txt");
+            try (FileOutputStream fileOutputStream = new FileOutputStream(file)) {
+              fileOutputStream.write(customHostNames.getBytes(StandardCharsets.UTF_8));
+            } catch (IOException e) {
+              e.printStackTrace();
+            }
+            task.systemProperty("angela.java.opts", "-Djdk.net.hosts.file=" + file.getAbsolutePath() + " -Xms64m -Xmx512m -DIGNITE_UPDATE_NOTIFIER=false");
+            task.systemProperty("jdk.net.hosts.file", file.getAbsolutePath());
+            task.systemProperty("angela.additionalLocalHostnames", customHostNames);
+          } else {
+            task.systemProperty("angela.java.opts", "-Xms64m -Xmx512m -DIGNITE_UPDATE_NOTIFIER=false");
```

Comment:
is it me or there are 2 `angela.java.opts` ? ;-)

Reply:
its an else block for scenario where customHosts are not configured